src/main/java/edu/wpi/first/gradlerio/GradleRIOPlugin.java

```diff
@@ -55,14 +55,10 @@ interface Parameters extends FlowParameters {
         }
 
         @Override
-        public void execute(Parameters parameters) {
+        public void execute(Parameters parameters) throws OneDriveException {
```

Comment:
No. Don't make this throw OneDriveException. Change OneDriveException to inherit from RuntimeException instead of just Exception.